app/assets/stylesheets/application.scss

```diff
@@ -18,14 +18,17 @@ Nav - header - #DAF2DC
 #4D5198 - text
 #FFCCE7 - button --- button:hover - #DAF2DC
  */
+
+@import "variables";
```

Comment:
usually all imports are declared at the top. However because bootstrap is being imported after the `*` selector where everything is getting zero-ed out, it seems that some stylings might have been created on how css was computed with this order